src/Output/Copy/Request/CookieJar.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoclerLabs\ApiClientGenerator\Output\Copy\Request;
+
+use InvalidArgumentException;
+use Psr\Http\Message\RequestInterface;
+
+class CookieJar
```

Comment:
As we don't use Guzzle anymore, let's use the psr7 implementation for cookies:
https://github.com/php-http/message/blob/master/src/Cookie.php
https://github.com/php-http/message/blob/master/src/CookieJar.php

Reply:
hmm psr/http-message (PSR) doesn't have it... the link is actually from a PSR7 implementation (like guzzle/psr7) which adds extra objects like this Cookie/CookieJar which is not PSR (still). Do you think it's worth it anyway instead of using simple CookieJar object? If it was PSR I would change it just fine... but I'm not sure adding another ps7 implementation is good...

Reply:
PSR always contains only the interfaces.
This is preferable because it is an official PHP implementation. Although, maybe you are right, and the fewer dependencies client would have the better.